packages/backend/src/request/requestInfo.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { App } from '@h4bff/core';
+import { RequestInfo } from './requestInfo';
+import { Request, Response } from 'express';
+
+describe('RequestInfo', () => {
+  it('#_setRequestResponse should set request and response', () => {
```

Comment:
prefer tests for public API if possible

Reply:
Even though we annotated the method as an `internal` one, technically it is part of the public API, that's why I added test for it.